src/main/java/Storage.java

```diff
@@ -10,69 +10,71 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class FileManipulator {
+public class Storage {
     protected final static String FOLDERPATH = java.nio.file.Paths.get(System.getProperty("user.dir"), "data").toString();
-    protected final static String FILEPATH = System.getProperty("user.dir").endsWith("text-ui-test")
-            ? java.nio.file.Paths.get(System.getProperty("user.dir"), "data", "duke.txt").toString()
-            : java.nio.file.Paths.get(System.getProperty("user.dir"), "data", "duke.txt").toString();
+    protected final static String FILEPATH = java.nio.file.Paths.get(System.getProperty("user.dir"), "data", "duke.txt").toString();
```

Comment:
there should be underscores to separate words for constant names, eg. FOLDER_PATH and FILE_PATH